src/components/SearchContainer/SearchContainer.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import ToggleComponent from '../ToggleComponent/ToggleComponent';
+import SearchBar from '../SearchBar/SearchBar';
+import './SearchContainer.scss';
+
+const SearchOptions = {
+    first: 'Title',
+    second: 'Genre'
+};
+
+export default class SearchContainer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            error: false,
+        };
+        this.sendSearchRequest = this.sendSearchRequest.bind(this);
+    }
+
+    sendSearchRequest(value) {
+        if (value === 'error') {
+            this.setState({ error: true });
+        } else {
+            const { handleSearchValue, } = this.props;
+            this.setState({ error: false });
+            handleSearchValue(value.toLowerCase());
+        }
+    }
+
+    render() {
+        const { error, } = this.state;
+        const {
+            toggleSearch,
+            searchOption,
+        } = this.props;
+
+        if (error) {
+            throw new Error('I crashed!');
```

Comment:
why application could crash?

Reply:
I implemented an error: enter "error" into search input. This is done just to enforce an error to see how it works